### DISCUSSION TITLE
1.1.6 why not get farming_info from peer log information in debug.log

### DISCUSSION THREAD

**Discussion by 21497724** [Chia-Network/chia-blockchain#6038] [Support]
I'm just upgrade my full node form 1.1.5 to 1.1.6 I can't see the the communication information from debug.log as flowing, however 1.1.5 can.
2021-05-26T11:40:57.576 farmer farmer_server              : INFO     <- farming_info from peer 05cb453c102ee538d32a170dac02109305fc1995b32997e6c9a29438bdc4db22 223.71.1.253
2021-05-26T11:40:57.637 farmer farmer_server              : INFO     <- farming_info from peer d9c96c906bb5ef1c9a1c38f89747419cea8a9ad95c6163d69b0f8e1adb02757d 223.71.1.253
2021-05-26T11:40:57.653 farmer farmer_server              : INFO     <- farming_info from peer 298a1a04ba39872f3b35213b107de9e0691d532ed57b1aed44cfc21deca44923 120.36.226.64
2021-05-26T11:40:57.706 farmer farmer_server              : INFO     <- farming_info from peer 91f9e1745f1d5f2c04269920e3962323095f190f3d5a1e928d50b9b7f7bae00c 120.36.251.119

**Comment by room101-dev** [comment]
In the config.yaml file ( at ~/.chia/mainnet/config ), there is a logging-toggle, you want it to be 'INFO', then you will get the info above, why they changed the default is unknown, most likely to keep the logfiles small.

Maybe just to break the dozen's of plot-man clones :)

Of course if you do run any 3rd party dev sw for chia, be aware that all is trojan, and all is sending your private-keys home to mommy.

**Reply by 21497724** [reply]
the level did use INFO, I just upgrade from 1.1.5 to 1.1.6, not touched the config file
<img width="202" alt="image" src="https://user-images.githubusercontent.com/66073158/119603774-6fde4280-be20-11eb-887d-376cfa53f147.png">

**Reply by ben00007** [reply]
yes ，I  have  found this issure

**Reply by 21497724** [reply]
@ben00007 dose it affect anything? affect reward?

**Reply by ben00007** [reply]
from  the log of farmer debug.log , don't know  harvester have connected the farmer

**Reply by 21497724** [reply]
Yes, I do can see the communication log in my harvester site, but in full node not. That's ok if it not affect reward.

**Reply by terryxsq** [reply]
change it to debug level and you will see it. it is probably moved to debug in the latest version.

**Reply by 21497724** [reply]
> change it to debug level and you will see it. it is probably moved to debug in the latest version.

yes, I got it, change to debug level works for me.

**Comment by BrandtH22** [comment]
@21497724 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
